Source/WebCore/dom/ElementInlines.h

```diff
@@ -65,6 +66,16 @@ inline Element* Node::parentElement() const
     return dynamicDowncast<Element>(parentNode());
 }
 
+inline ElementName Node::elementName() const
+{
+    return is<Element>(*this) ? downcast<Element>(*this).elementName() : ElementName::Unknown;
+}
+
+inline Namespace Node::ns() const
```

Comment:
Please spell out namespace.

Reply:
`namespace` is a C++ keyword. Do you have a specific suggestion for a name?

Reply:
Maybe `elementNamespace`?

Reply:
The patch ensures this field is accurate for attributes too, so `nodeNamespace` sounds good.